### PR TITLE
[UI] Make UI consistent

### PR DESCRIPTION
**Pull request by MUzairS15**
**Description**

This PR 
1. Adds Undeploy in Filters
2. Deployment modal in Grid view
3. Changes for adding Validation support
4. Fix false positive Snackbar, when operation is unsuccessful
**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
While the Validation and Deployment modals will be individually available to users, so too, will they be shown together. Perhaps, use the same menu tab style for headings like Environment irrespective of whether Validation is displayed or not. - - reuse the same component.

File: ui/components/MesheryAdapterPlayComponent.js [hunk 1]
```diff
@@ -422,10 +425,13 @@ class MesheryAdapterPlayComponent extends React.Component {
         self.setState({ namespaceError : true });
         return;
       }
+      const operationName = selectedOp.replaceAll("_", " ").split(" ").map(word => word.charAt(0).toUpperCase() + word.slice(1)).join(' ');
+      console.log(operationName, "PPPPPPPPP", deleteOp, cat);
```
Comment (leecalcote) [inline ui/components/MesheryAdapterPlayComponent.js:429]: Remove

File: ui/components/MesheryAdapterPlayComponent.js [hunk 2]
```diff
@@ -442,7 +448,7 @@ class MesheryAdapterPlayComponent extends React.Component {
     } = this.state;
     const { adapter } = this.props;
     // const fileInput = document.querySelector('#k8sfile') ;
-
+    console.log("QWERTY", selectedOp);
```
Comment (leecalcote) [inline ui/components/MesheryAdapterPlayComponent.js:451]: remove

File: ui/components/MesheryApplications.js [hunk 3]
```diff
@@ -258,6 +268,7 @@ function MesheryApplications({
    */
   // ASSUMPTION: APPLICATION FILES ARE ONLY K8S MANIFEST
   const handleDeploy = (application_file) => {
+    console.log(application_file, "SHAIKH");
```
Comment (leecalcote) [inline ui/components/MesheryApplications.js:271]: remove

File: ui/components/MesheryApplications.js [hunk 4]
```diff
@@ -237,15 +242,20 @@ function MesheryApplications({
...
-      application_file : null
+      application_file : null,
+      name : "",
+      count : 0
     });
   }
 
-  const handleModalOpen = (app_file, isDeploy) => {
+  const handleModalOpen = (app_file, name, isDeploy) => {
+    console.log(typeof (app_file), "SHAIKH");
```
Comment (leecalcote) [inline ui/components/MesheryApplications.js:252]: remove

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/components/Filters.js [hunk 5]
```diff
@@ -296,6 +294,53 @@ function MesheryFilters({ updateProgress, enqueueSnackbar, closeSnackbar, user,
     );
   }
 
+  const handleDeploy = (filter_file) => {
+    console.log(filter_file, "QWERTYUZAIR");
```
[the remark this answers is not in the document]
Comment (MUzairS15) [reply ui/components/Filters.js]: O

File: ui/components/MesheryAdapterPlayComponent.js (hunk 1 above)
Comment (MUzairS15) [reply ui/components/MesheryAdapterPlayComponent.js]: O

File: ui/components/MesheryAdapterPlayComponent.js (hunk 2 above)
Comment (MUzairS15) [reply ui/components/MesheryAdapterPlayComponent.js]: P

File: ui/components/MesheryApplications.js (hunk 3 above)
Comment (MUzairS15) [reply ui/components/MesheryApplications.js]: S

File: ui/components/MesheryApplications.js (hunk 4 above)
Comment (MUzairS15) [reply ui/components/MesheryApplications.js]: 😅

File: ui/components/Filters.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (MUzairS15) [reply ui/components/Filters.js]: done

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**